packages/blocks/src/image-block/image-block.ts

```diff
@@ -22,18 +23,26 @@ export class ImageBlockComponent extends LitElement {
   @query('.resizable-img')
   _resizeImg!: HTMLElement;
 
+  @state()
+  _source!: string;
   // disable shadow DOM to workaround quill
   createRenderRoot() {
     return this;
   }
 
   // This is the initial width before event resize is applied
 
-  override firstUpdated() {
+  override async firstUpdated() {
     this.model.propsUpdated.on(() => this.requestUpdate());
     this.model.childrenUpdated.on(() => this.requestUpdate());
     // exclude padding and border width
     const { width, height } = this.model;
+    // FIXME: after export global storage, we can use it directly
+    const storage = new BlobStorage();
+    const provider = await IndexedDBBlobProvider.init('test');
```

Comment:
We should use workspace id here.

Reply:
This is just a temporary solution， I will fix it when exporting global storage.

Reply:
We already have `this.host.page.id` to use here. A storage store should be under page level, instead of workspace level.